src/modules/tools/spindle/SoftSerial/SoftSerial_tx.cpp

```diff
@@ -0,0 +1,87 @@
+/* 
+ * SoftSerial by Erik Olieman
+ * Date: 05 Jul 2014
+ * Revision: 10:236fce2e5b8c
+ * URL: http://developer.mbed.org/users/Sissors/code/SoftSerial/
+ */
+
+#include "SoftSerial.h"
+
+int SoftSerial::_putc(int c)
+{
+    while(!writeable());

```

Comment:
also not acceptible. this hangs the entire system killing virtually everything. ON-IDLE must be called withing this loop